### PR TITLE
Tweak nav alignment, responsiveness on homepage + about heroes

### LINKED ISSUE #394: [Idea]: Reduce the container width to 1200px
### Describe your idea / enhancement

Present Container Width i,e,1536px
image
Suggested Container Width of 1200px
image

### How would this idea / enhancement be beneficial?

I just feel reducing the container width would make it better looking since things are in the center which human eyes prefer to look at !

### PR DESCRIPTION
**Pull request by MattIPv4**
## Describe your changes

Resolves #394 by adjusting the alignment of the logo in the nav, such that the standard container edge is visually aligned with the main body of the logo rather than the true edge of the logo. This also tweaks the wrapping of the homepage hero and the about alveus/staff heroes.

Closes #396 

## Notes for testing your change

Homepage hero + about Alveus hero look good at all viewport sizes.

### CODE REVIEW

**Comment by MattIPv4** [comment]
@pjeweb lmk what you think of this visually, and lmk if there are any other pages you I think I should take another pass over.

File: apps/website/src/pages/about/alveus.tsx [hunk 1]
```diff
@@ -448,8 +448,8 @@ const AboutAlveusPage: NextPage = () => {
           </p>
         </div>
 
-        <div className="max-w-2xl basis-full p-4 pt-8 md:mx-0 md:basis-1/2 md:pt-4">
-          <Lightbox>
+        <div className="basis-full p-4 pt-8 xl:mx-0 xl:basis-1/2 xl:pt-4">
+          <Lightbox className="mx-auto max-w-2xl">
```
Comment (pjeweb) [inline apps/website/src/pages/about/alveus.tsx:452]: Not sure about the mx-auto for larger viewports. Maybe only center it on the smaller breakpoints? In any case this the staff page should probably get the same treatment.

**Review by pjeweb — COMMENTED** [review]
Looks great. The only thing is the video sitting differently on About Alveus and Staff page.

File: apps/website/src/pages/about/alveus.tsx (hunk 1 above)
Comment (MattIPv4) [reply apps/website/src/pages/about/alveus.tsx:452]: What viewport looks odd to you with mx-auto? At larger viewports, there is very little margin that I can see and the mx-auto doesn't make much difference?

Comment (pjeweb) [reply apps/website/src/pages/about/alveus.tsx:452]: For example at 1920 width. Tbh it does not really matter much. The site is great as it is. I mostly notice it in comparison with the staff page.

<table>
<tr>
 <td>

 <td>

<tr>
 <td>Staff page, right aligned
 <td>About Alveus page, centered
</table>

Looking at it, maybe you could add the left padding to the other hero sections while you are at it?

Comment (MattIPv4) [reply apps/website/src/pages/about/alveus.tsx:452]: 👍 I'll play around and try to get them all consistent

File: apps/website/src/components/layout/navbar/Navbar.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const Navbar = () => {
             {/* Logo */}
             <Link
               href="/"
-              className="flex items-center px-1 md:px-2"
+              className="-ml-4 flex items-center"
```
Comment (pjeweb) [inline apps/website/src/components/layout/navbar/Navbar.tsx:26]:
```suggestion
              className="flex items-center md:-ml-4"
```

this breaks on smartphones 
negative-margin-mobile

Comment (MattIPv4) [reply apps/website/src/components/layout/navbar/Navbar.tsx:26]: Oops, completely forgot this is shared across mobile + desktop!

**Review by pjeweb — APPROVED** [review]
thank you, looks good

**Pull request merged**